config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -288,3 +288,7 @@ presets:
   # instead of relying on the deprecated one from k/k repository.
   - name: USE_KUBEKINS_LOG_DUMPING
     value: "yes"
+
+- labels:
+    preset-e2e-scalability-periodics-master: "true"
+  env:
```

Comment:
Hmm - it seems that it has already been enabled before, right?

Reply:
I mean for master branch...

Reply:
Yep, that was this other PR you were mentioning

Reply:
https://github.com/kubernetes/test-infra/pull/19663